src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -69,6 +81,39 @@ def app_init(
     )
 
 
+@app.command("list-templates", hidden=True)
+def app_list_templates(**options) -> CommandResult:
+    """
+    Prints information regarding templates that can be used with snow app init.
```

Comment:
Nit: "... regarding the official templates ...", so it is clear that only our Github templates are listed in this command.